docker/dependencies/docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2019 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+---
+version: '3'
+services:
+  kafka-zookeeper:
+    image: openzipkin/zipkin-kafka
+    container_name: kafka-zookeeper
+    ports:
+      - 2181:2181
+      - 9092:9092
+      - 19092:19092
+  kafka-topics:
+    image: openzipkin/zipkin-kafka
+    container_name: kafka-topics
+    entrypoint: "/busybox/sh -c"
+    command: "
+      \"/busybox/sh /kafka/bin/kafka-run-class.sh kafka.admin.TopicCommand --zookeeper kafka-zookeeper:2181 --create --topic zipkin-spans --partitions 2 --replication-factor 1 --if-not-exists
+      &&
+      /busybox/sh /kafka/bin/kafka-run-class.sh kafka.admin.TopicCommand --zookeeper kafka-zookeeper:2181 --create --topic zipkin-trace --partitions 2 --replication-factor 1 --if-not-exists
+      &&
+      /busybox/sh /kafka/bin/kafka-run-class.sh kafka.admin.TopicCommand --zookeeper kafka-zookeeper:2181 --create --topic zipkin-dependency --partitions 2 --replication-factor 1 --if-not-exists\"
+      "
+    depends_on:
+      - kafka-zookeeper
+  zipkin:
+    image: openzipkincontrib/zipkin-storage-kafka
+    container_name: zipkin
+    hostname: zipkin # required to route call to scatter-gather endpoint properly. should not be needed after #40 is solved
+    ports:
+      - 9411:9411
+      - 9412:9412
```

Comment:
put a comment on this port everywhere :P

Reply:
Will be removed hopefully soon after #58 
Will require a new zipkin patch version with armería fixes